bin/utils/descriptions/generate_processing_page.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from flask.json import htmlsafe_dumps
+from jinja2 import Environment, FileSystemLoader
+from sqlalchemy.orm import load_only
+
+# Used only by imports in the next block
+from indico.modules.events.sessions.models.blocks import SessionBlock
+from indico.modules.events.contributions.models.references import SubContributionReference
+from indico.modules.events.sessions.models.persons import SessionBlockPersonLink
+from indico.modules.rb.models.holidays import Holiday
+
+from indico.web.flask.app import make_app
+
+from indico.core.db import DBMgr, db
+from indico.core.db.sqlalchemy.util.session import update_session_options
+from indico.modules.categories.models.categories import Category
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.contributions.models.subcontributions import SubContribution
+from indico.modules.events.models.events import Event
+
+
+def main():
+    events = (Event.query
+              .filter(Event.description.op('~')('</[a-zA-Z]+>'))
```

Comment:
`</[a-zA-Z]+\s*/?>` to handle xhtml-style tags - same for other occurrences of this regex (not sure if the postgres regex engine knows `\s`, otherwise just use plain ` `)